src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php

```diff
@@ -114,25 +116,28 @@ public function formatBatch(array $records)
      */
     public function format(array $record)
     {
+        $levelColor = self::$levelColorMap[$record['level']];
+
         $record = $this->replacePlaceHolder($record);
 
-        $levelColor = self::$levelColorMap[$record['level']];
+        $context = $extra = '';
+        if (!empty($record['context']) || !$this->options['ignore_empty_context_and_extra']) {
```

Comment:
we should not ignore an empty context if extras are not empty. Otherwise, you won't be able to know it is the extras (it will look like as if it is the context being dumped)